### DISCUSSION TITLE
About changing sounds

### DISCUSSION THREAD

**Discussion by bubugian** [XCSoar/XCSoar#1374] [General]
Reading the manual (7.0) it seems to be possible to change sounds:

```
 3.9 Acoustic alert and sound feedback
 XCSoar generates sounds for different events, and can be configured
 to have custom sounds for any event. See Section 14.13 for details on
 customisation.
```

In my opinion, Section 14.13, it's insufficient to understand how to change sounds.
For example, I'd like to trigger the message "airspace.wav" when entering in an airspace or a message "height_insufficient.wav" if my height became insufficient to reach next task. Is it possible ? How ?

**Comment by kobedegeest** [comment]
Yes that is possible. 
The airspace one is easy. You need to make a input file (.xci) with 

mode=default
type=gce
event=PlaySound("airspace.wav")
data=AIRSPACE_ENTER

For the heigth you will need some lua script. The developers manual is a bit more detailed about lua and input events. But you can make a small script that activates automaticly and checks every xx seconds to see if heigth went from sufficient to insufficient and then trigger the event playsound: xcsoar.fire_legacy_event("PlaySound", "height_insufficient.wav")

**Comment by lordfolken** [comment]
Any contribution to the dev manual on lua is very welcome. https://github.com/XCSoar/XCSoar/blob/master/doc/lua.rst

**Reply by kobedegeest** [reply]
lua is probably better documented than the input event system at the moment. I was planning on improving the input event documentation (as i like this system alot) can have a look at lua docs after that.

But the original post mainly refered to part 14.13 Status in the manual. Which is indeed not documented things that are missing there are file type extension, how to trigger these alerts where to save them. 

And honestly i am not sure this even is a feature? If it is I also don't know how to use it. If you know where in the code to look for this functionality you can let me know.
